EncounterMe/EncounterMe/Views/LoadRoutePage.xaml.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xamarin.Essentials;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace EncounterMe.Views
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class LoadRoutePage : ContentPage
+    {
+
+        IEnumerable<MapPin> _mapPins;
+        public LoadRoutePage(IEnumerable<MapPin> mapPins)
+        {
+            InitializeComponent();
+            _mapPins = mapPins;
+            RoutesListView.ItemsSource = GetSortedMapPins();
+        }
+
+        private IEnumerable<MapPin> GetSortedMapPins()
+        {
+            var objectsQueryOrderedByDistance = _mapPins.OrderBy(pin => pin.DistanceToUser);
```

Comment:
Prieš šitą tikriausiai reiktų dar kviesti CalculateDistances, nes dabar neapskaičiuoja atstumų tai mes kaip ir darom OrderBy pagal neegzistuojančias reikšmes?

Reply:
Good point :D pamiršau iškviesti aš ją :D